### DISCUSSION TITLE
Airflow scheduler gets exit while running application

### DISCUSSION THREAD

**Discussion by tanya151** [apache/airflow#40730] [General] [answered]
### Apache Airflow version

2.9.2

### What happened?

I created a simple dag on my local MAC machine by installing apache airflow, using SqlLite as my database. Initially all configuration was set, web server and scheduler both were running fine, but I accidently closed the terminal on my Pycharm terminal. Now everytime it throws me error - Connection in use ('::',8793) Can't Connect to ('::',8793). 

I tried uninstalling airflow but it didn't work. I changed the database to PostgresSQL, webserver works fine but scheduler keeps throwing the same error.
<img width="1280" alt="Screenshot 2024-07-11 at 17 04 50" src="https://github.com/apache/airflow/assets/175310486/3b9fdc9d-fd1a-471e-bcd6-97fcb2bd64cc">
<img width="1280" alt="Screenshot 2024-07-11 at 17 07 14" src="https://github.com/apache/airflow/assets/175310486/86815140-1454-4135-8b43-5e60396ebaa1">
<img width="1135" alt="Screenshot 2024-07-11 at 17 08 06" src="https://github.com/apache/airflow/assets/175310486/a759a6d2-f2f1-4be6-88a4-7b3f55b35799">

### What you think should happen instead?

Why is this port not getting unblocked?

### How to reproduce

Close the terminal on Pycharm after running airflow scheduler command and this error comes up.

### Operating System

MAC OS - Intel

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
In the future - please avoid opening issues on airflow when you have trouble with your environment. Issues are for reporting issues with Airflow not issues with your environment that you want help with. Use discussions instead.

My guess is that you have process stil running somewhere and yoy need to kill it (google it if needed). Also maybe restarting your IDE will help. This is nothing specific to Airflow it's just process running that you need to learn how to deal with when you are developing and running development code.

**Reply by tanya151** [reply]
Hi Potiuk,

Thank you for the information, I will start a discussion rather than reporting it is an issue. 

Also, regarding the above issue, there are two parts:
1. Connection in use('::',8793) - you are correct this got resolved after restarting the laptop
2. 'No log template entry found for ID None, please make sure you set up your metabase correctly'. - This issue still persists.

Below configuration setting are done by me in airflow.cfg file, it will be helpful if you help me resolve the issue.

sql_alchemy_conn =postgresql+psycopg2://airflow_user:airflow_pass@localhost:5432/airflow_db
sql_alchemy_schema = airflow_schema
executor = LocalExecutor

**Reply by potiuk** [reply]
No idea about that one. But I suggest to open another discussion and describe it there. The problem when you are trying to put several different things in the issue is that if somoene who knows about something and see that they can help, will likely not be able to help on all of the issues. You are now implicitly assuming that I will continue helping with that, but people here help when they see something they generally know the answer on without too much of their FREE time spent. so you cannot assume that all your problems will be helped by the same person. This is a free forum, where you **might** get help  but you are not guaranteed. So I suggest you separate that other thing to another discussion - and hope somene will be able to help .

**Comment by tanya151** [comment, marked as answer]
Hi All,

The issue got resolved for PostgresSQL by running below SQL commands on PostgresSQL, mentioned in Airflow documentation:
<img width="851" alt="image" src="https://github.com/user-attachments/assets/a4ce00da-d685-4082-b8b6-adb93f88f45e">

Additional command required is to set the search path to the schema in which you want to store airflow metastore tables- ALTER USER airflow_user SET search_path = public;

Below configuration setting helped resolved the issue(removed port from conn URL):
sql_alchemy_conn =postgresql+psycopg2://airflow_user:airflow_pass@localhost/airflow_db
sql_alchemy_schema = public
executor = SequentialExecutor
